CONTRIBUTING.md

```diff
@@ -21,11 +21,13 @@ See [GitHub's official documentation](https://help.github.com/articles/using-pul
 
 # Getting Started
 
-- TODO: include build steps here.
+This repo contains source code, package and testing materials showing how to test the functionality of these measurements.
+
+See mroe details of `Getting Started` in the [README](/README.md).
```

Comment:
typo - "more" and let's rephrase to "See more details _for_ `Getting Started` in the [...]"